code/__DEFINES/station.dm

```diff
@@ -4,6 +4,7 @@
 
 
 #define STATION_TRAIT_ABSTRACT (1<<0)
+#define STATION_TRAIT_NORANDOM (1<<1)
```

Comment:
make sure to documetn these

Reply:
Whew okay, how should I document these exactly? Just saying what it's for? (Innate Station Traits)